src/tests/utils.cairo

```diff
@@ -18,21 +18,35 @@ fn deploy(contract_class_hash: felt252, calldata: Array<felt252>) -> ContractAdd
 
 /// Pop the earliest unpopped logged event for the contract as the requested type
 /// and checks there's no more data left on the event, preventing unaccounted params.
-/// This function also removes the hashed event-name key to support indexed event
-/// members.
+/// This functions also removes the first key from the event. This is because indexed
```

Comment:
```suggestion
/// This function also removes the first key from the event. This is because indexed
```
or did you mean
```suggestion
/// These functions also remove the first key from the event. This is because indexed
```
?

Reply:
The former